cmd/integration_test/build/buildspecs/tinkerbell-test-eks-a-cli.yml

```diff
@@ -37,6 +37,12 @@ env:
     T_TINKERBELL_IMAGE_UBUNTU_1_25: "tinkerbell_ci:image_ubuntu_1_25"
     T_TINKERBELL_IMAGE_UBUNTU_1_26: "tinkerbell_ci:image_ubuntu_1_26"
     T_TINKERBELL_IMAGE_UBUNTU_1_27: "tinkerbell_ci:image_ubuntu_1_27"
+    T_TINKERBELL_IMAGE_UBUNTU_2204_1_22: "tinkerbell_ci:image_ubuntu_2204_1_22"
```

Comment:
1.22 isn't supported anymore. We don't need to include this

Reply:
Ahh I just copied the existing Ubuntu 2004 vars which included 1-22, I'll remove both.